Multisig/UI/App/UIKit/TransactionlListViewController/TransactionListViewController.swift

```diff
@@ -89,6 +100,16 @@ class TransactionListViewController: LoadableViewController, UITableViewDelegate
         }
     }
 
+    func asyncTransactionList(address: Address, completion: @escaping (Result<Page<SCG.TransactionSummaryItem>, Error>) -> Void) -> URLSessionTask? {
+        // Should be overrided in subclass
```

Comment:
I think you can make result not optional and put here some preconditionFailure

Reply:
the thing is ClientGatewayService is also return URLSessionTask? 
so if I make it not optional here then I need to force unwrap the result which makes the code not safe IMO.